.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11","3.12"]
+        python-version: ["3.9", "3.10", "3.11","3.12"]
```

Comment:
Would it hurt add 3.13?

Reply:
I haven't really tested it..but there was a conflict earlier

Reply:
Fair enough